hydra-plutus/src/Hydra/Contract/Head.hs

```diff
@@ -271,8 +276,12 @@ checkClose ::
   ContestationPeriod ->
   Bool
 checkClose ctx headContext parties initialUtxoHash snapshotNumber closedUtxoHash sig cperiod =
-  checkSnapshot && mustBeSignedByParticipant ctx headContext
+  hasBoundedValidity
+    && checkSnapshot
+    && mustBeSignedByParticipant ctx headContext
  where
+  hasBoundedValidity = traceIfFalse "hasBoundedValidity check failed" $ tMax - tMin <= cp
```

Comment:
is it possible for `tMin` to be greater than `tMax`, or that is checked somewhere else?

Reply:
You can't post a tx that doesn't have the correct validity I think. You could omit the lower/upper bound but if you set it I would assume that it would get rejected by the node. We can write a test to prove it!

Reply:
added a mutation test for this case also